pkg/gcpclient/mock/client_generated.go

```diff
@@ -7,8 +7,8 @@ package mock
 import (
 	gomock "github.com/golang/mock/gomock"
 	gcpclient "github.com/openshift/hive/pkg/gcpclient"
-	compute "google.golang.org/api/compute/v1"
-	dns "google.golang.org/api/dns/v1"
+	v1 "google.golang.org/api/compute/v1"
```

Comment:
I don't get any of the changes in this file when I run `make generate`. Is there some difference in the gomock version being used?

Reply:
❯ /home/dgoodwin/go/bin/mockgen -version
v1.4.4

I tried removing my binary and re-running make generate but it brought the same version back. What version is on your end?

Reply:
That is the version that I have to and matches the version from the go.mod file.

Reply:
I can't seem to make it revert, tried removing and regenerating as well.

Reply:
Can we keep this change out of this PR since it is not related to the rest of the PR? If we determine that the change is needed, then we can add it in its own PR?

Reply:
Done.